From 0a6de048e9bbb6ec980af16feb3b875aa139128f Mon Sep 17 00:00:00 2001
From: Jaroslav Mracek <jmracek@redhat.com>
Date: Wed, 3 Jul 2019 12:56:31 +0200
Subject: [PATCH] Revert consequences of Fail-Safe mechanism

---
 libdnf/dnf-rpmts.cpp | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

--- a/libdnf/dnf-rpmts.cpp
+++ b/libdnf/dnf-rpmts.cpp
@@ -69,11 +69,9 @@ test_fail_safe(Header * hdr, DnfPackage
             DnfSack * sack = dnf_package_get_sack(pkg);
             auto includes = dnf_sack_get_module_includes(sack);
             if (!includes || !includes->has(dnf_package_get_id(pkg))) {
-                g_set_error(error, DNF_ERROR, DNF_ERROR_INTERNAL_ERROR,
-                            _("No available modular metadata for modular package '%s'; "
-                              "cannot be installed on the system"),
-                            dnf_package_get_nevra(pkg));
-                ret = FALSE;
+                g_critical(_("No available modular metadata for modular package '%s'; "
+                             "cannot be installed on the system"),
+                           dnf_package_get_nevra(pkg));
             }
         }
     }
